docs/conf.py

```diff
@@ -166,6 +166,10 @@ def setup(app):
     "install-nvidia-device-plugin",
     "tag/System/operation/SystemAuth",
     "batch-size-related-parameters",
+    "metrics",
+    "join-metrics",
+    "exposed-metrics",
+    "L139-L169",
```

Comment:
it seems like sphinx finds github anchors problematic in general, let's just ignore a glob/regex for all github anchors? these individual line links are going to be especially hard to catch if we slightly change the link url.

there's `linkcheck_anchors_ignore_for_url`, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_anchors_ignore_for_url which seems a good fit for our use case?

Reply:
This is a good idea, but I don't know that it's the right fit here. 

`linkcheck_anchors_ignore_for_url` requires sphinx version 7.1+, but `docs /requirements.txt` pins sphinx to `sphinx==4.2.0`

I also think the brittleness of specifying specific anchors is a good thing in this case. Not all github links will necessarily be valid; shouldn't we check them one by one and manually "ok" them?